src/spatialCursor/index.js

```diff
@@ -92,10 +93,53 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
         
         addSpatialCursor();
         addTestSpatialCursor();
-        toggleDisplaySpatialCursor(false);
+        toggleDisplaySpatialCursor(DEFAULT_SPATIAL_CURSOR_ON);
 
         // begin update loop
         update();
+        
+        const ADD_SEARCH_TOOL_WITH_CURSOR = false;
+        
+        if (ADD_SEARCH_TOOL_WITH_CURSOR) {
+            document.addEventListener('pointerdown', (e) => {
+                if (!indicator2 || !indicator2.visible) return;
+                if (realityEditor.device.isMouseEventCameraControl(e)) return;
+                if (!realityEditor.device.utilities.isEventHittingBackground(e)) return; // if clicking on a button, etc, don't trigger this
+
+                // raycast against the spatial cursor
+                let intersects = realityEditor.gui.threejsScene.getRaycastIntersects(e.clientX, e.clientY, [indicator2]);
+                if (intersects.length > 0) {
+                    addToolAtScreenCenter('searchDigitalThread');
+                }
+            });
+        }
+    }
+
+    // copied from pop-up-onboarding addon – should we refactor somewhere convenient to both?
```

Comment:
It looks like realityEditor.spatialCursor.addToolAtScreenCenter would be a good place